### PR TITLE
Gulp Babili issue

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ options was not passed properly.. 
+ But still couldn't figure out why the comments are removed for the previous case though..

### CODE REVIEW

**Comment by vigneshshanmugam** [comment]
rebase screw up fixing it..

**Comment by boopathi** [comment]
Think you forked from #395 branch

**Comment by hzoo** [comment]
You can run `git fetch upstream master && git fetch rebase master`

**Comment by vigneshshanmugam** [comment]
Done :) thanks guys

File: packages/gulp-babili/__tests__/gulp-babili-test.js [hunk 1]
```diff
@@ -148,9 +148,9 @@ describe("gulp-babili", () => {
       });
     });
 
-    xit("should take a custom function", () => {
+    it("should take a custom function", () => {
       return new Promise((resolve, reject) => {
-        const stream = gulpBabili({
+        const stream = gulpBabili({}, {
```
Comment (vigneshshanmugam) [inline packages/gulp-babili/__tests__/gulp-babili-test.js]: this is the options miss I was talking about @boopathi

File: packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap [hunk 2]
```diff
@@ -1,11 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`gulp-babili comments should remove all comments when false 1`] = `"foo(),bar(),baz();"`;
+exports[`gulp-babili comments should remove all comments when false 1`] = `"function foo(){}bar();var a=baz();"`;
```
Comment (boopathi) [inline packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap]: converting `foo()` to `function foo() {}` is the weirdest thing to get it working

Comment (vigneshshanmugam) [inline packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap]: Yes lol.. Dont even know how its working.. Just want to let you guys know there is something wrong!

Comment (boopathi) [inline packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap]: So, when converting multiple statements to sequence expressions, the comments are dropped. Should add a test case that this doesn't affect the surrounding block comments.

Comment (vigneshshanmugam) [inline packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap]: Yes exactly.. add a separate test case?

**Pull request merged**